### DISCUSSION TITLE
Inputting trawl survey location data

### DISCUSSION THREAD

**Discussion by annepolyakov** [sdmTMB/sdmTMB#357] [General]
Hello! I had a quick question about inputting trawl survey location data. Since trawl surveys are conducted along a transect, which location along the transect is typically used for input into the spatiotemporal model? For example, in the Pacific Cod example using the sdmTMB package, do the locations refer to the starting/ending/middle location along the trawl? Thank you!

**Comment by Lewis-Barnett-NOAA** [comment]
Yep, ideally the middle if you can calculate it accurately, or start/end.

On Thu, Jul 18, 2024 at 10:04 AM annepolyakov ***@***.***>
wrote:

> Hello! I had a quick question about inputting trawl survey location data.
> Since trawl surveys are conducted along a transect, which location along
> the transect is typically used for input into the spatiotemporal model? For
> example, in the Pacific Cod example using the sdmTMB package, do the
> locations refer to the starting/ending/middle location along the trawl?
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pbs-assess/sdmTMB/discussions/357>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKMJP4FSEXRO3TEDJM3EELZM7YQTAVCNFSM6AAAAABLDCOI72VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHE2TAMZXGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Lewis Barnett, PhD (he/him)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4111

**Comment by seananderson** [comment]
Yes. And the custom is usually region-specific, I think. That Pacific Cod dataset is likely using the starting coordinates. One hopes that the estimated correlation range is large enough that this choice doesn't make too big of a difference.

**Reply by annepolyakov** [reply]
Thank you so much, that is really helpful!
